app/controllers/notes_controller.rb

```diff
@@ -2,8 +2,8 @@ class NotesController < ApplicationController
   before_action :check_logged_in
   before_action :set_note, only: [:show, :edit, :update, :destroy]
   def index
-    @notes_by_month = Note.page(params[:page]).per(10).grouped_by_month(current_user)
-    @notes = Note.where(user: current_user).page(params[:page]).per(10)
```

Comment:
And after checking your `application_controller` - I doubt you have to implement `check_logged_in` yourself. As far as I know, devise has a helper for that.

And have you heard about **authorization** along with **authentication**? The keywords are `cancancan` and `pundit`.